src/privateStore.js

```diff
@@ -38,14 +43,16 @@ class PrivateStore {
    */
   async set (key, value) {
     if (!this.db) throw new Error('_sync must be called before interacting with the store')
+    if (key === SALT_KEY) throw new Error('Invalid key')
```

Comment:
I guess technically the key `SALT_KEY` could be allowed because the real DB key would be `Hash(salt + "SALT_KEY")`. But I agree it's less confusing to just treat it as off-limits.

Reply:
Oh, good point. I'll just remove it from here then. I guess this code could cause more confusion than it helps.